.gitignore

```diff
@@ -1,2 +1,3 @@
 # PyCache files
 __pycache__/
+cutlass_library.egg-info/
```

Comment:
what does this line do?

Reply:
`cutlass_library.egg-info` is generated in the source folder and do we need it? I don't think we need this folder to be added to GitHub so I added to `.gitignore`. 

```bash
ls -l python/cutlass_library.egg-info/
total 16
-rw-rw-r-- 1 manigupta manigupta 167 Oct 10 18:41 PKG-INFO
-rw-rw-r-- 1 manigupta manigupta 550 Oct 10 18:41 SOURCES.txt
-rw-rw-r-- 1 manigupta manigupta   1 Oct 10 18:41 dependency_links.txt
-rw-rw-r-- 1 manigupta manigupta  16 Oct 10 18:41 top_level.txt
```